test/packages/apache/manifest.yml

```diff
@@ -1,7 +1,7 @@
 format_version: 1.0.0
 name: apache
 title: Apache
-version: 0.1.4
+version: 0.2.7
```

Comment:
Is this change on purpose?

Reply:
Yes, I had to make it because the package registry image loaded up by the tests installs Apache version 0.2.6. So, in order for the asset test runner to be able to install the local apache package (i.e. the one being tested), I had to ensure that the local apache package's version was greater than the one in the registry. Otherwise the Install Package API would not allow the local package to be loaded up.

I admit that this is not a very maintainable change. Every time the package registry image is refreshed with a newer version of `apache`, we will need to bump up the version here as well. Maybe we should just set it here to something like `999.999.999`?

Reply:
Oh, I see :) In this case you can set it to sth weird like 999. (at least the major). Nice catch!